src/sentry/static/sentry/app/views/organizationContext.jsx

```diff
@@ -155,14 +158,26 @@ const OrganizationContext = createReactClass({
           GlobalSelectionStore.loadInitialData(data, this.props.location.query);
         }
         OrganizationEnvironmentsStore.loadInitialData(environments);
-
-        this.setState({
-          organization: data,
-          loading: false,
-          error: false,
-          errorType: null,
-          hooks,
-        });
+        this.setState(
+          {
+            organization: data,
+            loading: false,
+            error: false,
+            errorType: null,
+            hooks,
+          },
+          () => {
+            // Take a measurement for when organization details are done loading and the new state is applied
+            metric.measure({
+              name: 'app.component.organization-details-fetch',
+              start: 'organization-details-fetch-start',
+              data: {
+                route: getRouteStringFromRoutes(this.props.routes),
+                organization_id: data.id,
```

Comment:
We should cast this to an int to be consistent with how track org ids elsewhere (this can also be important in reload, but not in this case)